specs/merge/fork-choice.md

```diff
@@ -71,16 +59,14 @@ def finalize_block(self: ExecutionEngine, block_hash: Hash32) -> bool:
 ### `TransitionStore`
 
 ```python
-@dataclass
```

Comment:
Don't understand why these are not simply SSZ containers?

Reply:
It's not supposed to be an SSZ container. Btw, it's removed in https://github.com/ethereum/consensus-specs/pull/2605. We should probably wait for the resolution on this PR

Reply:
For things we don't serialize and instead are just used to represent requisite data stored in the consensus client to peform the operations in some way, we tend to use `@dataclass`. When we instead use an SSZ container, it signals that this data will need to be serialized and/or hashed in specific ways to conform to spec.

See the fork choice spec for the `Store` as an example

But, it is being removed in #2605 so resolving this